specification/devcenter/DevCenter/Environments/models.tsp

```diff
@@ -9,6 +9,69 @@ using TypeSpec.Http;
 
 namespace DevCenterService;
 
+@doc("Type of the output value")
+@added(APIVersions.v2024_02_01)
+union EnvironmentOutputType {
+  @doc("The output is an array of values.") 
+  array: "array",
+
+  @doc("The output is a boolean value.") 
+  boolean: "boolean",
+
+  @doc("The output is a number value.") 
+  number: "number",
```

Comment:
I noticed in the examples that we accept type "int" but it's not described in here

Reply:
Was this a mistake in previous swagger?

Reply:
Spoke with Jared offline, here is here answer:

"
Yeah we had a discussion on this some time back. The real issue is with ParameterType.number and integer we had modeled our input params on
[json schema](https://json-schema.org/understanding-json-schema/reference/type)
. We realized when we first implemented outputs that [ARM](https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/data-types)
, [Terraform](https://developer.hashicorp.com/terraform/language/expressions/types)
and other deployment tools don't tend to split these concepts, so we really have no use at all for both an integer and a number concept. For now, we decided to merge the 2 concepts in outputs to avoid this problem.

So short answer: yeah it's expected.
"